manifest.json

```diff
@@ -30,6 +30,18 @@
                 "name": "domain",
                 "type": "domain"
             },
+            {
+                "name": "username",
+                "type": "user"
+            },
+            {
+                "name": "email",
+                "type": "email"
```

Comment:
That question is not a standard one, so you should add an `ask` key to specify what's the point of this question

Note that you could also choose to fetch the email corresponding to the admin user previously chosen. You can do so using `email=$(ynh_user_get_info "$admin" 'mail')` inside the install script

Reply:
Updated.